app/controllers/course/admin/admin_controller.rb

```diff
@@ -11,6 +11,15 @@ def update #:nodoc:
     end
   end
 
+  def destroy #:nodoc:
```

Comment:
The latest push solves this problem, but because I didn't change this particular line, the comment still stays. 

Other than this, the only other thing I can extract is `@course.errors.full_messages.to_sentence` - into a single call. I see that happening a lot over most controllers, so maybe I can do that in a separate PR if it's possible? Otherwise, I'll just leave it as it is.